Nos/Views/Note/NoteView.swift

```diff
@@ -61,7 +64,44 @@ struct NoteView: View {
     }
     
     var note: Event
-    
+
+    /// Retrieves a list of authors associated with the current note and sorts them 
+    /// by the number of mutual followees with the current author.
+    ///
+    /// - Returns: A sorted array of `Author` objects, in descending order of mutual followees with the current author.
+    private func getSortedAuthorsByMutualFollowees() -> [Author] {
+        var authors: Set<Author> = []
+        guard let currentAuthor = currentUser.author else { return Array(authors) }
+
+        // Include the author of the root note
+        if let rootNoteAuthor = note.rootNote()?.author {
+            authors.insert(rootNoteAuthor)
+        }
+
+        // Include the author of the current note
+        if let noteAuthor = note.author {
+            authors.insert(noteAuthor)
+        }
+
+        // Include authors referenced in the current note
+        let noteAuthorReferences = note.loadAuthorsFromReferences(in: persistenceController.viewContext)
+        authors.formUnion(noteAuthorReferences)
+
+        // Include authors referenced in the root note
+        if let rootNoteAuthorReferences = note.rootNote()?.loadAuthorsFromReferences(
+            in: persistenceController.viewContext
+        ) {
+            authors.formUnion(rootNoteAuthorReferences)
+        }
+
+        // Include up to 10 authors from direct replies
+        let tenDirectReplies = directReplies.compactMap { $0.author }.prefix(10)
```

Comment:
I know we wanted to do this to minimize the performance impact, but I wonder whether this does? It seems like we already have all the directReplies, so including them in the authors we return probably wouldn't affect performance? Unless I'm missing something. And maybe limiting it to 1000 or something would still be a good idea.

Reply:
After discussing on a call, I'm good with this as-is. I think we'll want to try it out anyway and see if we want to change this number.